docs/release-notes/3219-force-pull-image.txt

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**New Features**
+
+-  The ``environment.force_pull_image: true`` behavior has improved when running on Determined
+   Agents. Previously, it would attempt delete the image if it existed before re-pulling it. This
```

Comment:
this plus, i could take or leave the "this behavior was wasteful ..." sentence
```suggestion
   Agents. Previously, it would delete the image if it existed before re-pulling it. This
```